benchexec/tools/jbmc.py

```diff
@@ -0,0 +1,148 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+Copyright (C) 2007-2015  Dirk Beyer
+All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+import xml.etree.ElementTree as ET
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+    Tool info for JBMC (http://www.cprover.org/cbmc/).
+    It always adds --xml-ui to the command-line arguments for easier parsing of
+    the output, unless a propertyfile is passed -- in which case running under
+    SV-COMP conditions is assumed.
```

Comment:
This seems quite similar to CBMC. Is there the possibility to inherit from the tool-info module from CBMC and reduce the duplicate code?